### PR TITLE
Support Imperial units

### PR DESCRIPTION
**Pull request by mrose17**
Fixes https://github.com/naofireblade/homebridge-weather-plus/issues/18

Plus a little jshint cleanup!

### CODE REVIEW

**Comment by naofireblade** [comment]
That's awesome! I will merge it asap.

File: util/characteristics.js [hunk 1]
```diff
@@ -26,8 +28,69 @@ const inherits = require('util').inherits,
...
+          , perms   : [Characteristic.Perms.READ, Characteristic.Perms.NOTIFY]
+          }, range);
+    };
+    var visibilityValue = (val) => {
+      return (units === 'metric') ? val : km2mi(val);
+    };
+
+    var kmh2mih = (km) => { return (km / 1.60934); };
+    var windspeedProps = (max) => {
+        var range = (units === 'metric') ? { unit: 'k/h', maxValue: max,          minValue: 0 }
```
Comment (naofireblade) [inline util/characteristics.js]: Can you change this to 'km/h'?

**Comment by mrose17** [comment]
good catch, thanks!

**Comment by naofireblade** [comment]
I added you to the contributors, please tell me if that's (un)desirable for you before I merge it into the trunk.

**Comment by mrose17** [comment]
@naofireblade - thanks. that works for me!

**Comment by mrose17** [comment]
also, after doing the merge, can you bump the version number and `npm publish`? thanks!

**Comment by naofireblade** [comment]
I think something went wrong with your merge. I already merged the trunk into this branch. You tried to revert it but some things are still not reverted. I will go back to revision 577470d.

**Comment by mrose17** [comment]
sorry about that. i accidentally pulled from upstream(!!) it loos good to me.

**Comment by naofireblade** [comment]
Published to npm in v2.2.0
https://www.npmjs.com/package/homebridge-weather-plus/v/2.2.0

**Pull request merged**